docs/application/dotnet/guides/app-management/event.md

```diff
@@ -0,0 +1,291 @@
+# Event Broadcast and Subscription
+
+
+The application can broadcast its own events to all listeners who want to listen. The events can either be predefined [system events from the platform](#platform) or user-defined events. Platform modules can broadcast system events whereas UI and service applications broadcast user-defined events.
```

Comment:
@divyk As I asked, would you please check the sentence "The events can either be predefined system events from the platform or user-defined events."?

Reply:
No problem for technical meaning